types/src/language_storage.rs

```diff
@@ -22,7 +23,10 @@ pub enum TypeTag {
     Struct(StructTag),
 }
 
-#[derive(Serialize, Deserialize, Debug, PartialEq, Hash, Eq, Clone, PartialOrd, Ord)]
+#[derive(
+    Serialize, Deserialize, Debug, PartialEq, Hash, Eq, Clone, PartialOrd, Ord, CryptoHasher,
+)]
+#[CryptoHasherSalt = "VM_ACCESS_PATH"] // fga: is this what we want? See ModuleId
```

Comment:
Who is the best person to answer that one? if nobody, rename fga -> TODO

Reply:
Done!